docs/UserGuide.md

```diff
@@ -43,33 +43,51 @@ simpler to use while being even more functional than alternatives on the market.
 
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer. If not, you can download Java `17` [here](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html).
+### Step 1: Check if Your Computer is Ready
```

Comment:
Also, do you think it'll value add if we put these sub-headers into the table of contents as well?